schwifty/registry.py

```diff
@@ -41,7 +45,12 @@ def has(name: str) -> bool:
 def get(name: str) -> Union[Dict, List[Dict]]:
     if not has(name):
         data = None
-        directory = files(__package__) / f"{name}_registry"
+        package_path: Union[Traversable, Path]
+        if getattr(sys, "frozen", False) and hasattr(sys, "_MEIPASS"):
+            package_path = Path(getattr(sys, "_MEIPASS"))
+        else:
+            package_path = files(__package__)
```

Comment:
PyInstaller maintainer here: Wherever you got this code snippet from, can you please delete/downvote/mark it as invalid/expunge it from the internet. It is misinformation. 

PyInstaller expects you to preserve package structure (i.e. these data files would go into `$app_root/schwifty/bank_registry` instead of `$app_route/bank_registry`). Without this change, a user could have used `--collect-data=schwifty` and everything would have worked out fine. With this change, the user has to force their JSON files into the wrong place.